core/app/services/spree/export/optional_attributes.rb

```diff
@@ -0,0 +1,19 @@
+module Spree
+  module Export
+    class OptionalAttributes
```

Comment:
I think this SO should be included in `google` namespace.  Not sure if we be able to use this SO in other places ( outside export to google ) if not, imho, we shouldn't leave that without proper namespace

Reply:
@nciemniak thoughts/ideas ?

Reply:
Yeah, it will probably only work for google since the attribute names probably differ between companies. Would it make sense to actually move all of them to google namespace, including rss generation?

Reply:
I would keep those services under one namespace, to have this feature consistent 👌🏻